### PR TITLE
Update Example with the latest API

### PR DESCRIPTION
**Pull request by juliocamarero**

### CODE REVIEW

**Comment by gamerson** [comment]
```
:blade.controlmenuentry:compileJava/home/travis/build/gamerson/liferay-blade-samples/liferay-gradle/blade.controlmenuentry/src/main/java/com/liferay/blade/samples/controlmenuentry/BladeProductNavigationControlMenuEntry.java:48: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
 FAILED
```

@juliocamarero perhaps we need to update to new API?

**Comment by juliocamarero** [comment]
I think this last commit should fix it.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

File: liferay-gradle/blade.controlmenuentry/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compile group: "com.liferay", name: "com.liferay.product.navigation.control.menu.api", version: "2.0.0"
+	compile group: "com.liferay", name: "com.liferay.product.navigation.control.menu.api", version: "[3.0.0-SNAPSHOT,3.0.0)"
```
Comment (gamerson) [inline liferay-gradle/blade.controlmenuentry/build.gradle]: Hey @juliocamarero i don't see 3.0.0 available here:
https://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.product.navigation.control.menu.api/

So I hesitate to update this sample until that version is available.  what do you think?

Comment (juliocamarero) [inline liferay-gradle/blade.controlmenuentry/build.gradle]: The Jar is published now, I just added another commit removing the usage of the snapshot.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/38. [See changes here.](https://github.com/gamerson/liferay-blade-samples/compare/5c3c743~2...5c3c743) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

**Pull request closed without merging**